src/components/proofEditor/proofEditor.module.scss

```diff
@@ -0,0 +1,15 @@
+.proof-editor {

```

Comment:
You don't need an empty css class; and if you do need this for some reason, it should be called `proofEditor`.